runtime/Cpp/runtime/nuget/pack.cmd

```diff
@@ -0,0 +1,93 @@
+echo off
+rem echo Usage:
+rem echo ------
+rem echo pack vsvers version [pre]      // pack 2019 4.9.1 -beta
+rem echo ------
+setlocal enableextensions enabledelayedexpansion
+
+if "%1"=="" goto usage
+if "%2"=="" goto usage
+set PRE=%3
+set PLATFORM=Win32
+
+rem -version ^^[16.0^^,17.0^^)
+set VS_VERSION=vs%1
+rem  should be set "VSWHERE='%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe  -property installationPath -version ^[16.0^,17.0^)'"
+if %VS_VERSION%==vs2019 (
+  set "VSWHERE='C:\PROGRA~2\"Microsoft Visual Studio"\Installer\vswhere.exe  -latest -property installationPath -version ^[16.0^,17.0^)'"
+) else (
+if %VS_VERSION%==vs2022 (
+  set "VSWHERE='C:\PROGRA~2\"Microsoft Visual Studio"\Installer\vswhere.exe  -latest -property installationPath -version ^[17.0^,18.0^)'"
```

Comment:
Does it work with vs2022? It's x64 and it's being installed to `Program Files` unlike previous versions that are being installed to `Program Files (x86)`.

Reply:
Yes. wswhere.exe is always in Program File(x86). You can test it by running
```
C:\PROGRA~2\"Microsoft Visual Studio"\Installer\vswhere.exe  -latest -property installationPath -version ^[17.0^,18.0^)
```